add-on/_locales/en/messages.json

```diff
@@ -83,13 +83,13 @@
     "message": "Enable/disable all IPFS integrations on $1.",
     "description": "A menu item tooltip in Browser Action pop-up (panel_activeTabSiteIntegrationsToggleTooltip)"
   },
-  "panel_pinCurrentIpfsAddress": {
-    "message": "Pin IPFS Resource",
-    "description": "A menu item in Browser Action pop-up (panel_pinCurrentIpfsAddress)"
+  "panel_importCurrentIpfsAddress": {
+    "message": "Import to Files at My Node",
+    "description": "A menu item in Browser Action pop-up (panel_importCurrentIpfsAddress)"
   },
-  "panel_pinCurrentIpfsAddressTooltip": {
-    "message": "Pin this page's IPFS resources to your node to have a local copy that's available offline and never thrown away.",
-    "description": "A menu item tooltip in Browser Action pop-up (panel_pinCurrentIpfsAddressTooltip)"
+  "panel_importCurrentIpfsAddressTooltip": {
+    "message": "Import this tab's IPFS resource to your Files to have a persistent local copy that's available offline and never thrown away.",
```

Comment:
@jessicaschilling this is the only change in UI, lmk if the label and tooltip explainer could be improved :pray: 

> ![import-to-Files-2021-05-07--21-50-40](https://user-images.githubusercontent.com/157609/117504163-52852980-af82-11eb-8f68-53253f8d6fb7.png)

Reply:
@lidel Maybe "on My Node" rather than "at"?